pkgs/development/python-modules/typer/default.nix

```diff
@@ -53,12 +53,12 @@ buildPythonPackage rec {
   preCheck = ''
     export HOME=$(mktemp -d);
   '';
-  disabledTests = lib.optionals stdenv.isDarwin [
+
+  disabledTests = [
+    "test_install_completion"
```

Comment:
Tests should have a short comment why they are disabled.

Reply:
Added a comment.